tests/RobotFramework/tests/cumulocity/service_monitoring/service_monitoring.robot

```diff
@@ -144,6 +144,8 @@ Custom Setup
 
 Check if a service is up
     [Arguments]    ${service_name}
+    # Ensure tedge-mapper-c8y is running because it publishes messages to the cloud
+    ThinEdgeIO.Start Service    tedge-mapper-c8y
```

Comment:
Just to be as safe as possible.
```suggestion
    ThinEdgeIO.Start Service    tedge-mapper-c8y
    ThinEdgeIO.Service Should Be Running    tedge-mapper-c8y
```

Reply:
Though technically that does not offer any improvements as checking that status won't make the test pass more or less (as the following assertions are "smart" assertions and don't rely on the condition being immediately true).

Reply:
But I've added the suggestion as it like at least speed up the diagnosis phase if it does fail rather than digging through the logs to confirm that the mapper is actually running.